### PR TITLE
feat: devx major update 2 0 0

### PR DESCRIPTION
**Pull request by Kerwood**
- Changed port from 80 to 8080 on deployments to support .NET 8
- Updated iam-service-account to 2.0.0 which now supports Workload Identity
- Removed db-migration dep and implemented it into the chart to gain more control.
- Refactored the values.yaml file.
- Added hardened security settings to deployments.
- Updated sqlinstance dep to 3.0.0
- Updated SQL Cloud Proxy to 2.8

### CODE REVIEW

File: charts/devx-dashboard/Chart.yaml [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
+  version: 2.0.0
   repository: https://ok-amba.github.io/config-connector-helm
   alias: iamServiceAccount
   condition: deadLetter.enabled
 - name: sqlinstance
-  version: 2.1.4
+  version: 3.0
   repository: https://ok-amba.github.io/config-connector-helm
   condition: deadLetter.enabled
-- name: db-migration
```
Comment (andreasuvoss) [inline charts/devx-dashboard/Chart.yaml:16]: Why has this been deleted?

Comment (andreasuvoss) [reply charts/devx-dashboard/Chart.yaml:16]: I see it was added  in one of the templates.

File: charts/devx-dashboard/values.yaml [hunk 2]
```diff
@@ -153,48 +144,40 @@ deadLetter:
     labels: {}
     annotations: {}
 
+  # TODO: Refactor below appsettings config to user Workload Identity and .global.ProjcetID
```
Comment (andreasuvoss) [inline charts/devx-dashboard/values.yaml:147]: TODO?

**Review by andreasuvoss — APPROVED** [review]

File: charts/devx-dashboard/Chart.yaml (hunk 1 above)
Comment (Kerwood) [reply charts/devx-dashboard/Chart.yaml:16]: Yes, is was integrated to make the chart less complex.

File: charts/devx-dashboard/values.yaml (hunk 2 above)
Comment (Kerwood) [reply charts/devx-dashboard/values.yaml:147]: Is it done ? :)

File: charts/devx-dashboard/templates/configmap-appsettings.yaml [hunk 3]
```diff
@@ -32,17 +32,17 @@ data:
...
         "ProdTestPipelines": {{ .Values.releaseBoard.pipelines.prodTest | toPrettyJson | nindent 10 }}
       },
       {{- end }}
       {{- if .Values.deadLetter.enabled }}
       "Database": {
-        "ConnectionString": "Host=localhost;Port=5432;Database={{ .Values.sqlinstance.name }};Username={{ .Values.iamServiceAccount.name }}@{{ .Values.googleProject.projectId }}.iam;Max Auto Prepare=100;"
+        "ConnectionString": "Host=localhost;Port=5432;Database={{ .Values.sqlinstance.name }};Username={{ .Values.iamServiceAccount.name }}@{{ .Values.global.projectID }}.iam;Max Auto Prepare=100;"
       },
       "PubSubDeadLetter": {
         "Emulator": false,
```
Comment (andreasuvoss) [inline charts/devx-dashboard/templates/configmap-appsettings.yaml:44]: Tilføj

`"UseWorkLoadIdentity": true,`

**Pull request merged**